devicetypes/smartthings/zigbee-window-shade.src/zigbee-window-shade.groovy

```diff
@@ -16,6 +16,7 @@ import physicalgraph.zigbee.zcl.DataType
 metadata {
 	definition(name: "ZigBee Window Shade", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.blind", mnmn: "SmartThings", vid: "generic-shade") {
 		capability "Actuator"
+        capability "Battery"
```

Comment:
nit: indentation is using spaces instead of tabs

Reply:
fixed